### ISSUE TITLE
SyntaxError: Identifier 'route' has already been declared

### ISSUE THREAD

**Issue by arturi** [yoshuawuyts/bankai#75]
Thanks for the tool! Stumbled into an error:

``` bash
~/ bankai start
/usr/local/lib/node_modules/bankai/bin/start.js:59
    const route = opts.html.entry || '/bundle.js'
    ^

SyntaxError: Identifier 'route' has already been declared
    at start (/usr/local/lib/node_modules/bankai/bin/start.js:27:16)
    at main (/usr/local/lib/node_modules/bankai/bin/index.js:143:3)
    at Object.<anonymous> (/usr/local/lib/node_modules/bankai/bin/index.js:146:1)
```

**Comment by marionebl** [comment]
You're running node@4, right?

Should be fixed when https://github.com/yoshuawuyts/bankai/pull/74 lands.

**Comment by arturi** [comment]
No, node v5.4.1, that’s why I decided to submit this anyway.

**Comment by marionebl** [comment]
Fixed with [v3.3.0](https://github.com/yoshuawuyts/bankai/releases/tag/v3.3.0)
